tests/ui/target-feature/issue-108680-1.rs

```diff
@@ -0,0 +1,12 @@
+// only-aarch64
```

Comment:
I recommended this to you earlier, but now I wonder if having aarch64 specific errors is a good idea. You are on M1 hardware, I am on AMD 64 bit hardware. Blessing these tests on another platform is not as easy.

If someone in the future wishes to change error messages, they either have to manually edit the stderr files or they have to find a way to build both for aarch64 and x86. Without aarch64 specific target_feature tests it would only be one target. On the other hand, it's always good to test platform specific features on multiple targets, and we don't know if aarch64 might also make its way into the world of non-mac developers. they will certainly appreciate having tests for their platform.

@compiler-errors what do you think?

Reply:
Yeah, I get it, so I'll wait for the answer then, meanwhile I'll keep these files.